07LessonSeven/homework/index.html

```diff
@@ -5,6 +5,28 @@
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-
+  <section>
+    <header>
+      <h1>Profession Portal</h1>
+    </header>
+    <nav>
+      <ul>
+        <li>Home</li>
+        <li>Companies</li>
+        <li>Positions</li>
+      </ul>
+    </nav>
+      <section class="bigbox">
+        <section class="box1">
+          <h1>Hire an employee</h1>
+          <p>Vivamus congue rhoncus sem et placerat. Fusce nec nunc lobortis lorem ultrices facilisis.Ut dapibus blandit nunc, et consectetur dui.</p>
+          <div>Sign up as company</div>
+        </section>
+        <section class="box2">
+          <h1>Looking for a job</h1>
+          <p>Vivamus congue rhoncus sem et placerat. Fusce nec nunc lobortis lorem ultrices facilisis.Ut dapibus blandit nunc, et consectetur dui.</p>
+        </section>
+      </section>
+  </section>
 </body>
 </html>
```

Comment:
will do


On Tue, Aug 16, 2016 at 6:37 PM, Justin Lugo <notifications@github.com>
wrote:

> In 07LessonSeven/homework/index.html
> <https://github.com/AustinCodingAcademy/intro-workbook/pull/50#discussion_r75040575>
> :
>
> > +        <li>Companies</li>
> > +        <li>Positions</li>
> > +      </ul>
> > +    </nav>
> > +      <section class="bigbox">
> > +        <section class="box1">
> > +          <h1>Hire an employee</h1>
> > +          <p>Vivamus congue rhoncus sem et placerat. Fusce nec nunc lobortis lorem ultrices facilisis.Ut dapibus blandit nunc, et consectetur dui.</p>
> > +          <div>Sign up as company</div>
> > +        </section>
> > +        <section class="box2">
> > +          <h1>Looking for a job</h1>
> > +          <p>Vivamus congue rhoncus sem et placerat. Fusce nec nunc lobortis lorem ultrices facilisis.Ut dapibus blandit nunc, et consectetur dui.</p>
> > +        </section>
> > +      </section>
> > +  </section>
> >  </body>
> >  </html>
>
> It looks like you were off to a good start for this assignment. You seem
> to have all the necessary content (except for the "Sign up as an employee"
> text) but the finished product is not done. Keep working on it and submit
> another commit when you are done.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/AustinCodingAcademy/intro-workbook/pull/50/files/a168f0b12d60784681d872eed815c0ec57580877..28419985b5e2f97b1766a004408a42e25ce2a78f#r75040575>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATfHyscG0vIbIDKgmyG4Ex7V1qD8dqhPks5qgkmcgaJpZM4JQVtS>
> .
>